README.md

```diff
@@ -1,6 +1,8 @@
 # `isolate()`
 
-> A utility function to make scoped dataflow components in Cycle.js.
+> A utility function to make scoped dataflow components in Cycle.js. For example, this isolates DOM events seen by a component to those from it's own nodes, not the entire tree. 
```

Comment:
Sure but I felt it need a bit of context as 'scoped dataflow components' is very abstract and I could not figure out what it did till someone pointed out the docs. And then I found the video. I came here from seeing Isolate in the examples and wanted to know what it was about.